tests/e2e/test-data/certs/client.key

```diff
@@ -0,0 +1,3 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
I think you could make the whole certs in one file. But this is also a good one.

Reply:
we decided to not put text to save a memory on CI (no matter what is inside this cert)